packages/shorebird_cli/lib/src/shorebird_config_mixin.dart

```diff
@@ -50,20 +53,24 @@ mixin ShorebirdConfigMixin on ShorebirdCommand {
     return Pubspec.parse(yaml);
   }
 
-  ShorebirdYaml addShorebirdYamlToProject(String appId) {
-    File(
-      p.join(Directory.current.path, 'shorebird.yaml'),
-    ).writeAsStringSync('''
+  ShorebirdYaml addShorebirdYamlToProject(AppId appId) {
+    const content = '''
 # This file is used to configure the Shorebird updater used by your application.
 # Learn more at https://shorebird.dev
 # This file should be checked into version control.
 
 # This is the unique identifier assigned to your app.
 # It is used by your app to request the correct patches from Shorebird servers.
-app_id: $appId
-''');
+app_id:
+''';
+    final yaml = File(p.join(Directory.current.path, 'shorebird.yaml'));
```

Comment:
Nit: this would be a tiny bit easier to follow if this were created after the `editor`, possibly as a one-liner:

```dart
final editor = YamlEditor(content)
  ..update(['app_id'], appId.value ?? appId.values);
File(p.join(Directory.current.path, 'shorebird.yaml')).writeAsStringSync(editor.toString());
```

Reply:
Sounds good to me, done ✅